app/mysql/init.js

```diff
@@ -0,0 +1,22 @@
+
+
+    con.query("CREATE DATABASE IF NOT EXISTS plan", function(err) {
+      if (err)
+        throw err;
+
+      con.query("USE plan", function(err) {
+        if(err)
+          throw err;
+        con.query("CREATE TABLE IF NOT EXISTS users("
+          + "id int NOT NULL AUTO_INCREMENT,"
+          + "email varchar(50) NOT NULL,"
+          + "pswd varchar(50) NOT NULL,"
+          + "PRIMARY KEY(id)"
+          + ")", function(err) {
+            if(err)
+              throw err;
+          });
+      });
+    });
+    
+}
```

Comment:
Originally, I wasn't getting connect.js to work with both creating the connection and checking for/setting up the database. I've deleted that file now, since I got it to work in connect.js, but it looks like I managed to do the deletion after I sent the pull request and didn't catch it. It doesn't affect functionality though, I didn't get far enough that anything depended on it.